swapserverrpc/go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/lightninglabs/loop/swapserverrpc
 
 require (
 	google.golang.org/grpc v1.56.3
-	google.golang.org/protobuf v1.30.0
+	google.golang.org/protobuf v1.33.0
```

Comment:
I think we'll also need to bump the `protobuf-go-hex-display` to 1.33 as well as need a few changes in our RPC generator dockerfile: https://github.com/lightninglabs/loop/commit/54937f132ce4e5c49ef679e5861a8f5f88d31240

Can prepare a PR to incorporate everything.

Reply:
Oh actually this is already done in master.

Reply:
The swapserverrpc change is not in master though.